person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(_id, age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_service
+    of_age || parent_permission == true
+  end
```

Comment:
- [ ] It is a code style convention Method "can_use_service" should include "?" at the end as follow "can_use_service?" that way we are telling that method to return just true or false.
- [ ] "parent_permission" should be called as an instance variable as follow @parent_permission without "== true" if not the variable does not be called.